src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/misc/StrangeNetherGoo.java

```diff
@@ -37,4 +43,18 @@ public int getBarteringLootChance() {
         return chance.getValue();
     }
 
+    @Nonnull
+    @Override
+    public ItemUseHandler getItemHandler() {
+        return this::use;
+    }
+
+    public void use(PlayerRightClickEvent e) {
+        if (e.getClickedBlock().isPresent()) {
+            if (e.getClickedBlock().get().getType().name().contains("SIGN")) {
```

Comment:
Use tags.

Reply:
Tag.SIGNS please

Reply:
Didn't know that was a thing, i will get on it sorry.